ui/README.md

```diff
@@ -105,26 +105,28 @@ setting `VAULT_UI` environment variable.
 
 ### Quick commands
 
-| Command                                                                                  | Description                                                         |
-| ---------------------------------------------------------------------------------------- | ------------------------------------------------------------------- |
-| `yarn start`                                                                             | start the app with live reloading                                   |
-| `yarn start:mirage <handler>`                                                            | start the app with the mocked mirage backend, with handler provided |
-| `make static-dist && make dev-ui`                                                        | build a Vault binary with UI assets, by default runs on port :8200  |
-| `ember g component foo --in lib/core`                                                    | generate a component in the /addon engine                           |
-| `echo "export { default } from 'core/components/foo';" > lib/core/app/components/foo.js` | export component from addon engine to main app                      |
-| `yarn test:quick -f='<test name>'` -s                                                    | run tests in the browser, filtering by test name                    |
-| `yarn lint:js`                                                                           | lint javascript files                                               |
+| Command                               | Description                                                             |
+| ------------------------------------- | ----------------------------------------------------------------------- |
+| `yarn start`                          | start the app with live reloading                                       |
+| `yarn start:mirage <handler>`         | start the app with the mocked mirage backend, with handler provided     |
+| `make static-dist && make dev-ui`     | build a Vault binary with UI assets (run from root directory not `/ui`) |
+| `ember g component foo -ir core`      | generate a component in the /addon engine                               |
+| `yarn test:quick -f='<test name>'` -s | run tests in the browser, filtering by test name                        |
+| `yarn lint:js`                        | lint javascript files                                                   |
```

Comment:
This is so helpful 😍 

Could we also add a quick command for generating an ember engine? It's also the ember engine docs, but could be helpful to have in our docs too! `ember g in-repo-engine <engine-name>`

Reply:
Ooh, maybe a whole section about how to create an engine would be helpful! Since it doesn't happen nearly as oftenn as these other items

Reply:
ohhh yes! That would be helpful for me and possibly for future new hires! ember engines are new to me 🙈